tests/admin_checks/tests.py

```diff
@@ -38,6 +38,7 @@ def check(cls, model, **kwargs):
         return ['error!']
 
 
+
```

Comment:
Only two empty lines required. You can run `flake8` on source to spot these kind of warnings.